src/msg/async/dpdk/DPDKStack.cc

```diff
@@ -242,11 +242,11 @@ int DPDKWorker::connect(const entity_addr_t &addr, const SocketOptions &opts, Co
   return r;
 }
 
-void DPDKStack::spawn_worker(unsigned i, std::function<void ()> &&func)
+void DPDKStack::spawn_worker(std::function<void ()> &&func)
 {
   // create a extra master thread
   //
-  funcs[i] = std::move(func);
+  funcs.push_back(std::move(func));
```

Comment:
I was wondering whether we needed to use std::forward() instead of std::move(), but I think the correct way is move() since the reference is being stored to the array of funcs.